docs(mongodb): wording + grammatical issues (#2111)
Scalingo/documentation: src/_posts/databases/mongodb/2000-01-01-start.md

## Changing Plans

> You can upgrade or downgrade your database plan whenever you need it. This operation happens with no manual input. The impact of changing the plan depends on what plans we are talking about:

Reviewer: **nitpick:** "we" is a bit unformal, and "what" is not well suited here

```suggestion
You can upgrade or downgrade your database plan whenever you need it. This operation happens with no manual input. The impact of changing the plan depends on which plans are involved:
```

> * When you change the amount of RAM available to your database, your database will be stopped then restarted on a new host with new parameters of the chosen plan. During the operation the connection is dropped between your app and the database. Finally, after the operation is successful, the related app will be restarted.

Reviewer: **nitpick:** Let's breathe a bit, and I think that we must not use the future?

```suggestion
* When you change the amount of RAM available to your database, your database will be stopped then restarted on a new host with new parameters of the chosen plan. During the operation, the connection is dropped between your app and the database. Finally, after the operation is successful, the related app is restarted.
```
